### ISSUE TITLE
[FEA] LocalCUDACluster should support passing a list of GPU IDs to use as workers

### ISSUE THREAD

**Issue by randerzander** [rapidsai/dask-cuda#56]
Users often share the same server or workstation with multiple GPUs. If they want to use LocalCUDACluster but don't want to use all GPUs (e.g. they're nice enough to leave some for others), they can currently specify n_workers.

However, on an 4 GPU system, if no CUDA_VISIBLE_DEVICES environment variables were set manually, and user1 creates a LocalCUDACluster with 2 workers, then user2 creates another LocalCUDACluster with 2 workers, both users will be using the same first two GPUs.

It would be nice if users could pass a list of GPU_IDs they want their LocalCUDACluster to use.

**Comment by mrocklin** [comment]
Agreed.  This is already in the dgx-1 branch.  Something like this should
be merged shortly.

**Comment by pentschev** [comment]
@mrocklin I think you're talking about #46, is that right? While I see that `CUDA_VISIBLE_DEVICES` is handled there, I don't think that's what this feature request is about. If I understand it correctly, what's being asked here is that `LocalCUDACluster` includes a parameter to say which GPUs should be used, `CUDA_VISIBLE_DEVICES` will still have precedence, but this new parameter would allow selecting which GPUs will be used by the `LocalCUDACluster` instance, and ignore all the others.

**Comment by mrocklin** [comment]
In that PR you can do the following:

```python
from dask_cuda import LocalCUDACluster

cluster = LocalCUDACluster(CUDA_VISIBLE_DEVICES=[2,3])
```

I *think* that this is the request above, but I could be wrong.

**Comment by pentschev** [comment]
You're right, I missed that `CUDA_VISIBLE_DEVICES` was an argument there. I would only suggest renaming that argument to something more pythonic, the current name could be misleading if you also define `CUDA_VISIBLE_DEVICES` when launching the python process, I would prefer to make a clear distinction between those two.

**Comment by pentschev** [comment]
This feature was merged today in branch-0.8, closing this issue.
